### PR TITLE
Add missing light colour defines

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Finishes converting light fixture colours to defines.

## Why It's Good For The Game

Can't sleep at night with half being defines and half not.

### CODE REVIEW

File: code/__DEFINES/colors.dm [hunk 1]
```diff
@@ -182,13 +181,39 @@
...
+/// Less saturated red. rgb(255, 50, 50)
+#define LIGHT_FIXTURE_RED "#FF3232"
+/// Did you know we have two vars for fire colour? rgb(255, 17, 0)
+#define LIGHT_FIXTURE_RED_FIRE "#FF1100"
+/// For emergencies only, no fires allowed! rgb(255, 78, 78)
+#define LIGHT_FIXTURE_EMERGENCY "#FF4E4E"
+/// Rave party or crime scene. rgb(167, 0, 255)
+#define LIGHT_FIXTURE_BLACKLIGHT "#A700FF"
+/// Why are you still partying, there's a fire! rgb(212, 0, 255)
+#define LIGHT_FIXTURE_BLACKLIGHT_FIRE "#D400FF"
```
Comment (LemonInTheDark) [inline code/__DEFINES/colors.dm:213]: These names are a bit weird, think it's better to describe the color rather then the particular usage, least that's how I've worked with it in the past.

**Comment by lessthnthree** [comment]
Okay.

It was getting late and I was running out of ways to describe slightly
different shades of red.

I'll work on the names.

**Comment by LemonInTheDark** [comment]
not got the energy? that's a real shame

**Comment by lessthnthree** [comment]
I'm not going to delete the branch, but I don't want to leave it sitting here for a week. One day(tm)

**Comment by LemonInTheDark** [comment]
o7 go with god

**Pull request closed without merging**